crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -355,6 +355,41 @@ fn handle_enclosed_comment<'a>(
         AnyNodeRef::ExprGenerator(generator) if generator.parenthesized => {
             handle_bracketed_end_of_line_comment(comment, source)
         }
+        AnyNodeRef::StmtReturn(_) => {
```

Comment:
Do we need to cover other statements like `raise` and `assert`?

```python
raise (
    "a"
    "b"  # belongs to `b`
)

assert False, (
    "a"
    "b"  # belongs to `b`
)
```

Reply:
No, only statements that use `FormatStatementsLastExpression`